pkg/chaincode/commit.go

```diff
@@ -46,10 +43,24 @@ func createCommitProposal(CCDefine CCDefine, id identity.SigningIdentity) (*peer
 		Collections:         CCDefine.CollectionConfigPackage,
 	}
 
-	argsBytes, err := proto.Marshal(args)
+	commitArgsBytes, err := proto.Marshal(args)
 	if err != nil {
-		return nil, err
+		return err
+	}
+
+	gw, err := gateway.New(connection, id)
+	if err != nil {
+		return err
+	}
+	defer gw.Close()
+
+	approveProposal, err := gw.GetNetwork(CCDefine.ChannelID).
```

Comment:
@bestbeforetoday , I am confused a bit here.
to use gateway to submit a new commit proposal, I referenced approve code.

Which I suppose:
- gateway client is able to get endorsement policy base on channel name, contract and proposal name.
- once we use the proposal from gateway client, hence we can just invoke Submit function which it starts with endorsement and finished with a new block.

But...
error happens for endorsement policy at  https://github.com/hyperledger/fabric-admin-sdk/actions/runs/3958263163/jobs/6779618766#step:7:135